inc/admin/class-admin.php

```diff
@@ -168,10 +169,11 @@ public function check_for_event_submissions(){
 							$cat_name = $new_line;
 							$cat_slug = $new_line;
 						}
-						
+
 						if ($i == 0) {
-							if ( $this->termAlreadyExists($rootCategories, 'name', $cat_name, 'id') ) {
-								$parent_id = $this->termAlreadyExists($rootCategories, 'name', $cat_name, 'id');
+							//use isset rather than separate method, simpler.
+							if ( isset($rootCategories[$cat_name]) ) {
```

Comment:
with wp_list_pluck data extraction, it simplifies the validation by allowing you to use isset() instead of writing a separate method.